swap/swap.go

```diff
@@ -199,15 +204,28 @@ func New(dbPath string, prvkey *ecdsa.PrivateKey, backendURL string, params *Par
 		factory,
 	)
 	// start the chequebook
-	if swap.contract, err = swap.StartChequebook(chequebookAddressFlag, initialDepositAmountFlag); err != nil {
-		return nil, err
-	}
-	availableBalance, err := swap.AvailableBalance()
-	if err != nil {
+	if swap.contract, err = swap.StartChequebook(chequebookAddressFlag); err != nil {
 		return nil, err
 	}
 
-	swapLog.Info("available balance", "balance", availableBalance)
+	// deposit money in the chequebook if desired
+	if !noDepositFlag {
```

Comment:
I really don't like much this negated variable names, which then are logically negated again...
Can we do this differently?

Reply:
I see what you mean. The problem is that by default, we want to prompt the user about whether he wants to deposit money in his chequebook. And he can tell Swarm not to prompt by means of this flag. An option would be to call it `zeroDeposit`, do you think that is better?

Note: we can not just give 0 als a value for `--swap-deposit`, as the default value of this parameter is already zero, hence, we cannot distinguish between the user giving 0 as an argument for the `--swap-deposit` flag, or the user not specifying the `--swap-deposit-flag` and getting the default value.

Reply:
I think I would like `skipDepositPrompt`

Reply:
Maybe then `skipDeposit`? Because with a `--swap-deposit` flag you also skip the prompt

Reply:
Agreed

Reply:
Changed to `skipDeposit`